Example/FinancialConnections Example/FinancialConnectionsUITests/FinancialConnectionsUITests.swift

```diff
@@ -71,4 +75,253 @@ final class FinancialConnectionsUITests: XCTestCase {
+        XCTAssertTrue(enableTestModeSwitch.waitForExistence(timeout: 60.0))
+        if (enableTestModeSwitch.value as? String) == "1" {
+            enableTestModeSwitch.tap()
+        }
+        
+        let showAuthFlowButton = app.buttons["Show Auth Flow"]
+        XCTAssertTrue(showAuthFlowButton.waitForExistence(timeout: 60.0))
+        showAuthFlowButton.tap()
+        
+        let consentAgreeButton = app.webViews.buttons["Agree"]
+        XCTAssertTrue(consentAgreeButton.waitForExistence(timeout: 120.0)) // glitch app can take time to load
+        consentAgreeButton.tap()
+        
+        // find + tap an institution; we add extra institutions in case
+        // they don't get featured
+        let institutionButton: XCUIElement?
+        let institutionTextInWebView: String?
+        let chaseInstitutionButton = app.webViews.buttons["Chase"]
+        if chaseInstitutionButton.waitForExistence(timeout: 10) {
+            institutionButton = chaseInstitutionButton
+            institutionTextInWebView = "Chase"
+        } else {
+            let bankOfAmericaInstitutionButton = app.webViews.buttons["Bank of America"]
+            if bankOfAmericaInstitutionButton.waitForExistence(timeout: 10) {
+                institutionButton = bankOfAmericaInstitutionButton
+                institutionTextInWebView = "Bank of America"
+            } else {
+                let wellsFargoInstitutionButton = app.webViews.buttons["Wells Fargo"]
+                if wellsFargoInstitutionButton.waitForExistence(timeout: 10) {
+                    institutionButton = wellsFargoInstitutionButton
+                    institutionTextInWebView = "Wells Fargo"
+                } else {
+                    institutionButton = nil
+                    institutionTextInWebView = nil
+                }
+            }
+        }
+        guard let institutionButton = institutionButton, let institutionTextInWebView = institutionTextInWebView else {
+            XCTFail("Couldn't find a Live Mode institution.")
+            return
+        }
+        institutionButton.tap()
+        
+        let prepaneContinueButton = app.webViews.buttons["Continue"]
+        XCTAssertTrue(prepaneContinueButton.waitForExistence(timeout: 60.0))
+        prepaneContinueButton.tap()
+        
+        // check that the WebView loaded
+        let institutionWebViewText = app.webViews
+            .staticTexts
+            .containing(NSPredicate(format: "label CONTAINS '\(institutionTextInWebView)'"))
+            .firstMatch
+        XCTAssertTrue(institutionWebViewText.waitForExistence(timeout: 120.0))
+        
+        let secureWebViewCancelButton = app.buttons["Cancel"]
+        XCTAssertTrue(secureWebViewCancelButton.waitForExistence(timeout: 60.0))
+        secureWebViewCancelButton.tap()
+        
+        let playgroundCancelAlert = app.alerts["Cancelled"]
+        XCTAssertTrue(playgroundCancelAlert.waitForExistence(timeout: 60.0))
+    }
+}
+
+extension XCTestCase {
```

Comment:
Some of the helpers below are not used, but I kept them since they may be useful to make the tests more reliable